pb/pb_hooks/webhook.pb.js

```diff
@@ -52,3 +55,26 @@ onRecordAfterCreateRequest(e => {
     console.log(e)
   }
 })
+
+// This is here as the torrents get updated first then the entry
+// and due to how we expand for the torrent data it will only
+// ever have one data set to pull from meaning the data is the same.
+const torrentHook = e => {
+  try {
+    const record = e.record
+    const hooks = $app.dao()?.findRecordsByFilter('hooks', 'event = \'update\'')
+    if (!hooks || !record) return
+    if (record.collection()?.name !== 'torrents') return
+    const store = $app.store()
+    for (const hook of hooks || []) {
+      if (!hook || hook.get('collection') !== 'entries') continue
+
+      store.set(record.get('id'), record.originalCopy())
```

Comment:
is this necessary? I feel like doing
```js
const preRecord = record.originalCopy()
const prebest = preRecord.get('isBest')
const postbest = record.get('isBest')
```
accomplishes the same?

Reply:
yeah, this is unnecessary, you can simply do inside a `onRecordBeforeUpdateRequest` hook:
```js
    const original = e.record?.originalCopy()
    const updated = e.record?.cleanCopy()
```